build-logic/convention-plugins/src/main/kotlin/social.androiddev.library.gradle.kts

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id("social.androiddev.library.common")
+}
+
+val libs = the<org.gradle.accessors.dm.LibrariesForLibs>()
+
+kotlin {
+    jvm("desktop")
```

Comment:
Please remove "desktop". I believe it actually doesn't do anything and is there just for assigning a name